x-pack/legacy/plugins/monitoring/public/components/metricbeat_migration/instruction_steps/elasticsearch/enable_metricbeat_instructions.js

```diff
@@ -178,60 +171,18 @@ export function getElasticsearchInstructionsForEnablingMetricbeat(product, _meta
 
   let migrationStatusStep = null;
   if (product.isInternalCollector || product.isNetNewUser) {
-    let status = null;
-    if (hasCheckedStatus) {
-      status = (
-        <Fragment>
-          <EuiSpacer size="m"/>
-          <EuiCallOut
-            size="s"
-            color="warning"
-            title={i18n.translate('xpack.monitoring.metricbeatMigration.elasticsearchInstructions.isInternalCollectorStatusTitle', {
-              defaultMessage: `We have not detected any monitoring data coming from Metricbeat for this Elasticsearch.
-              We will continuously check every {timePeriod} seconds in the background.`,
-              values: {
-                timePeriod: autoCheckIntervalInMs / 1000,
-              }
-            })}
-          />
-        </Fragment>
-      );
-    }
-
-    let buttonLabel;
-    if (checkingMigrationStatus) {
-      buttonLabel = i18n.translate('xpack.monitoring.metricbeatMigration.elasticsearchInstructions.checkingStatusButtonLabel', {
-        defaultMessage: 'Checking for data...'
-      });
-    } else {
-      buttonLabel = i18n.translate('xpack.monitoring.metricbeatMigration.elasticsearchInstructions.checkStatusButtonLabel', {
-        defaultMessage: 'Check for data'
-      });
-    }
-
     migrationStatusStep = {
       title: product.isNetNewUser ? statusTitleNewUser : statusTitle,
       status: 'incomplete',
       children: (
-        <Fragment>
-          <EuiFlexGroup alignItems="center">
-            <EuiFlexItem>
-              <EuiText>
-                <p>
-                  {i18n.translate('xpack.monitoring.metricbeatMigration.elasticsearchInstructions.statusDescription', {
-                    defaultMessage: 'Check that data is received from the Metricbeat'
-                  })}
-                </p>
-              </EuiText>
-            </EuiFlexItem>
-            <EuiFlexItem grow={false}>
-              <EuiButton onClick={checkForMigrationStatus} isDisabled={checkingMigrationStatus}>
-                {buttonLabel}
-              </EuiButton>
-            </EuiFlexItem>
-          </EuiFlexGroup>
-          {status}
-        </Fragment>
+        <EuiCallOut
+          size="s"
+          color="warning"
+          title={i18n.translate('xpack.monitoring.metricbeatMigration.elasticsearchInstructions.isInternalCollectorStatusTitle', {
+            defaultMessage: `We have not detected any monitoring data coming from Metricbeat for this Elasticsearch.
```

Comment:
Maybe: "...for this Elasticsearch [cluster]"?

Reply:
Yup, nice catch again. It should be `node`